merlin/systems/triton/__init__.py

```diff
@@ -150,9 +150,6 @@ def _convert_tensor(t):
     out = t.as_numpy()
     if len(out.shape) == 2:
         out = out[:, 0]
```

Comment:
Unrelated to this change: It's unclear to me why we'd want to remove dimensions from the input here

Reply:
This code has existed for a long time, and I think is related to the perennial inconsistency around list formats that has plagued the Merlin code base. Way back in the before times, sometimes you'd get a proper 1d array/tensor and sometimes you'd get a 2d array/tensor that only contained one row. The legacy serving code from NVT that Systems is based on and still trying to clean up and/or shed had all kinds of issues like this and mostly solved them by hacking around the inconsistent formats instead of standardizing.